packages/react-server/docs/why-triton.md

```diff
@@ -1,16 +1,16 @@
-# Triton
+# react-server
 
 React is an amazing library for client-side user interface, and it has the added benefit of being able to be rendered on the server, which is crucial for SEO, SEM, and user experience. However, server-side rendering in practice is significantly more complicated than just calling `React.renderToString` and piping out the result, especially if you want to make sure your site loads quickly in a mobile-first world.
 
-`Triton`:
+`react-server`:
 1. Smoothes out the common problems you run into with React server-side rendering.
 1. Encodes performance best practices into the framework, making it easy to build high performance websites by default.
 
 ## What we talk about when we talk about performance
 
-When we talk about performance in `Triton`, we're mostly talking about _perceived performance_. We believe that [WebPageTest](http://www.webpagetest.org/) is correct to focus on how much time it takes from the moment the user navigates the browser until the content above the fold is painted. There are other important perceived performance metrics (like time to first content painted, time to all content painted, and time to full interactivity), but in developing Triton, we have focused most on reducing time to above-the-fold paint. WebPageTest's [Speed Index](https://sites.google.com/a/webpagetest.org/docs/using-webpagetest/metrics/speed-index) is probably the best current measure of above-the-fold paint speed, as it takes into account how quickly the content is painted during load, not just the total time to full completion.
+When we talk about performance in `react-server`, we're mostly talking about _perceived performance_. We believe that [WebPageTest](http://www.webpagetest.org/) is correct to focus on how much time it takes from the moment the user navigates the browser until the content above the fold is painted. There are other important perceived performance metrics (like time to first content painted, time to all content painted, and time to full interactivity), but in developing react-server, we have focused most on reducing time to above-the-fold paint. WebPageTest's [Speed Index](https://sites.google.com/a/webpagetest.org/docs/using-webpagetest/metrics/speed-index) is probably the best current measure of above-the-fold paint speed, as it takes into account how quickly the content is painted during load, not just the total time to full completion.
```

Comment:
These lines are too long to get within-line change highlighting.  Would be nice to break them up at some point.